templates/telco/osm/index.html

```diff
@@ -0,0 +1,473 @@
+{% extends "telco/base_telco.html" %}
+
+{% block title %}OSM{% endblock %}
+
+{% block meta_description %}{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit#heading=h.mgu0wm7n08jm{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>
+          NFV Orchestration with Open Source MANO
+        </h1>
+        <p>
+          Automate deployment and operations of network functions, and benefit from reduced operational costs, by using an open source implementation of ETSI NFV MANO. As a key contributor to the Open Source MANO project, we are excited to help you on your NFV journey.
+        </p>
+        <p>
+          <a href="/telco/osm/contact-us" class="p-button--positive js-invoke-modal">Contact us to migrate today</a>
+        </p>
+      </div>
+      <div class="col-5 u-align--center u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/a2e4b89f-opensource-MANO.svg",
+          alt="",
+          width="370",
+          height="133",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered no-padding--bottom">
+  <div class="row">
+    <div class="col-6">
+      <h2>OSM</h2>
+      <h4>Open-source implementation of ETSI NFV MANO stack</h4>
+      <p>Allows Telecommunications Service Providers (TSPs) to move from traditional, legacy networking services to cloud-native network functions.</p>
+      <p>Fully aligned with ETSI NFV Information Models.</p>
+      <p>
+        <a href="/engage/telco-network-functions-whitepaper">Download the whitepaper&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center u-hide--small">
+    </div>
+    <hr class="p-separator" />
```

Comment:
Only cols are allowed to be direct children of rows, this isn't within the col div - could you take it out of the row it's in and put it in a `<div class="u-fixed-width">`?  (Same with the others please)

Reply:
Do you want some help with this? we can jump on a call to show you what I mean if you like

Reply:
Are you referring to the hr tag?